inductiva/tasks/run_simulation.py

```diff
@@ -21,6 +22,7 @@ def run_simulation(
     api_method_name: str,
     input_dir: pathlib.Path,
     computational_resources: Optional[types.ComputationalResources] = None,
+    provider_id: ProviderType = ProviderType.GCP,
```

Comment:
to be fair, this should be:

```suggestion
    provider_id: Optional[Union[ProviderType, str]] = ProviderType.GCP,
```